lib/adapters/webdriverio.js

```diff
@@ -30,6 +30,10 @@ WebDriverIOAdaptor.prototype = objectAssign({}, Browser, {
     webdriverio.getElementSize(selector)
         .then(function(size) {
 
+          if (size instanceof Array) {
```

Comment:
Use [`Array.isArray()`](http://devdocs.io/javascript/global_objects/array/isarray) instead. It's more readable, and apparently [safer](http://web.mit.edu/jwalden/www/isArray.html).